apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -1,34 +1,35 @@
-import { NgFor, NgIf } from '@angular/common';
-import { Component, Input } from '@angular/core';
-import { randStudent, randTeacher } from '../../data-access/fake-http.service';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
-import { ListItemComponent } from '../list-item/list-item.component';
+import { NgFor, NgTemplateOutlet } from '@angular/common';
+import {
+  Component,
+  ContentChild,
+  EventEmitter,
+  Input,
+  Output,
+  TemplateRef,
+} from '@angular/core';
 
 @Component({
   selector: 'app-card',
   templateUrl: './card.component.html',
   standalone: true,
-  imports: [NgIf, NgFor, ListItemComponent],
+  imports: [NgFor, NgTemplateOutlet],
+  styles: [
+    `
+      .card-container {
+        background-color: var(--card-background-color, lightblue);
+      }
+    `,
+  ],
 })
 export class CardComponent {
-  @Input() list: any[] | null = null;
-  @Input() type!: CardType;
-  @Input() customClass = '';
+  @Input() list: any[] = [];
```

Comment:
I tried to pass the type for the list through here by starting with 
```
export class CardCoponent<T> {
         @Input() list: T[] = []
```

Then defining a viewChild in the parent where T was defined as either `city[]`, `teacher[], etc...